src/lib/y2users/read_result.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2issues"
+
+module Y2Users
+  # Class to represent the result of reading the configuration
+  class ReadResult
```

Comment:
NP: the equivalent class in *network* is called [`ReadingResult`](https://github.com/yast/yast-network/blob/master/src/lib/y2network/reading_result.rb). Maybe we should use the same name here, what do you think?

Reply:
Well, to be honest, I prefer `ReadResult`, so I guess we should adapt network instead. :-)